### DISCUSSION TITLE
Expected Memories Behavior

### DISCUSSION THREAD

**Discussion by andriykrefer** [immich-app/immich#6385] [Q&A]
Hi, I haven't started using Immich "for real" yet. Before that I am testing its features and limitations.
One feature that caught my eyes is the Memories features, that I think it is one the coolest features of the "other app".

In the other app, I often get notifications showing photos of the same date, but on previous years, and that is very nostalgic.
First of all, is that what Immich mean by  its "Memories" features? Or is it another thing?

Second, I tried to test it, setting the date of a photo as the next day, just to see the features in action. I waited for the next day and got no notification in the app. It was supposed to the send a notification, or is it only showed on the web app if I happened to open it on the correct date? (By the way, I did not it on the web app either which the few test photos that I put there).

Thanks, and congratulations for the project, I am very excited to be able to use it!

**Comment by alextran1502** [comment]
Memories will show up if you have the photos/videos on the same date in the previous years.

**Reply by andriykrefer** [reply]
Does the App should send a notification about the memory?

**Reply by bo0tzz** [reply]
Not at the moment, no

**Reply by andriykrefer** [reply]
Ok, thanks for the response
